portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -12,13 +12,19 @@
   private final long timeStamp;
   private final String dateTime;
   private final long id;
+  private long likes; 
 
-  public Comment(String firstName, String lastName, String commentText, long timeStamp, String dateTime, long id) {
+  public Comment(String firstName, String lastName, String commentText, long timeStamp, String dateTime, long id, long likes) {
```

Comment:
Nit: Per the style guide, it is recommended to keep line-widths to 100 characters.

This has many advantages, consistency and ability to review side-by-side being primary ones (in my view).